cupy/core/_kernel.pyx

```diff
@@ -873,46 +882,44 @@ cdef class ufunc:
             raise TypeError('Wrong number of arguments for %s' % self.name)
 
         dev_id = device.get_device_id()
-        args = _preprocess_args(dev_id, args, False)
+        arg_list = _preprocess_args(dev_id, args, False)
         if out is None:
-            in_args = args[:self.nin]
-            out_args = args[self.nin:]
+            in_args = arg_list[:self.nin]
+            out_args = arg_list[self.nin:]
         else:
             if self.nout != 1:
                 raise ValueError('Cannot use \'out\' in %s' % self.name)
             if n_args != self.nin:
                 raise ValueError('Cannot specify \'out\' as both '
                                  'a positional and keyword argument')
 
-            in_args = list(args)
+            in_args = arg_list
             out_args = _preprocess_args(dev_id, (out,), False)
-            args += out_args
+        del arg_list
 
-        in_args = _copy_in_args_if_needed(in_args, out_args)
+        _copy_in_args_if_needed(in_args, out_args)
```

Comment:
Why you don't have to use its return value?
I can't immediately understand what this line is, because `_copy_in_args_if_needed` has no comment...